package_manifest.yaml

```diff
@@ -161,6 +161,7 @@ foreman_scl_packages:
     rubygem-responders: {}
     rubygem-rest-client: {}
     rubygem-retriable: {}
+    rubygem-rkerberos: {}
```

Comment:
Shouldn't it be under `scl_and_nonscl`?

Reply:
I suppose I can. I was mostly aiming to deliver this in the SCL as prep work.